### PR TITLE
 index.md

### PR DESCRIPTION
**Pull request by Ainatiana**
Fix #58
Hey dear reviewer!
We have made a new PR so please, have a look.
Feel free to mention if there's a mistake.

Thanks in advance.
Dainatiel

### CODE REVIEW

File: e/e-index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# List of artists starting with E
+
+- [Eddy Lejao](http://eddylejao)
+- [Ejema](http://ejema)
+- [Elidiot](http://elidiot)
+- [Elidyah](http://elidyah)
+- [Elodie](http://elodie)
+- [Eric manana](http://ericmanana)
+- [Esperance](http://esperance)
```
Comment (ndorfin) [inline e/e-index.md]: These should be without the `http://` and have `.md` at the end.

File: e/e-index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# List of artists starting with E
+
+- [Eddy Lejao](http://eddylejao)
+- [Ejema](http://ejema)
+- [Elidiot](http://elidiot)
+- [Elidyah](http://elidyah)
+- [Elodie](http://elodie)
+- [Eric manana](http://ericmanana)
```
Comment (ndorfin) [inline e/e-index.md]: Capital M?

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: e/e-index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# List of artists starting with E
```
Comment (ndorfin) [inline e/e-index.md]: Filename looks wrong

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: e/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# List of artists starting with E
+
+- [Eddy Lejao](eddylejao.md)
```
Comment (ndorfin) [inline e/index.md]: This filename needs a dash to replace the space, right?

File: e/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- [Ejema](ejema.md)
+- [Elidiot](elidiot.md)
+- [Elidyah](elidyah.md)
+- [Elodie](elodie.md)
+- [Eric Manana](ericmanana.md)
```
Comment (ndorfin) [inline e/index.md]: And again

**Review by ndorfin — CHANGES_REQUESTED** [review]

**Comment by Ainatiana** [comment]
Please, have a look at our file again.We have edited it.
Thanks.

**Review by ndorfin — APPROVED** [review]
:shipit:

**Pull request merged**